### PR TITLE
6581 observer to enumerator

### PR DESCRIPTION
**Pull request by DevneyHamilton**
Not sure about schema.rb updates.

### CODE REVIEW

File: app/models/broadcast.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def self.receivable_association
   WHICH_PHONE_OPTIONS = %w(main_only alternate_only both)
 
   # options for recipients
-  RECIPIENT_SELECTION_OPTIONS = %w(all_users all_observers specific)
+  RECIPIENT_SELECTION_OPTIONS = %w(all_users all_enumerators specific)
```
Comment (hooverlunch) [inline app/models/broadcast.rb]: I think we will need a migration for this one, no?

File: config/locales/en.yml [hunk 2]
```diff
@@ -1134,7 +1134,7 @@ en:
         both: "Both SMS and email"
     recipient_selection_options:
       all_users: "All users in mission"
-      all_observers: "All observers in mission"
+      all_enumerators: "All enumerators in mission"
```
Comment (hooverlunch) [inline config/locales/en.yml]: I think key changes will need to be made in all the translation files, right?

File: db/schema.rb [hunk 3]
```diff
@@ -124,11 +124,10 @@
     t.string "op", limit: 255
     t.uuid "option_node_id"
     t.integer "option_node_old_id"
-    t.uuid "questioning_id"
+    t.uuid "questioning_id", null: false
     t.integer "questioning_old_id"
-    t.uuid "ref_qing_id"
+    t.uuid "ref_qing_id", null: false
     t.integer "ref_qing_old_id"
-    t.string "role", null: false
```
Comment (hooverlunch) [inline db/schema.rb]: I think what's going on here is that your DB has been migrated for one of the condition branches that haven't been merged yet. Can you please remove this from the PR (ideally using rebase to keep the history clean). Then I think most of those test failures will go away.

File: config/locales/en.yml (hunk 2 above)
Comment (DevneyHamilton) [reply config/locales/en.yml]: I wasn't sure about this - how will translators for other languages know those ones should be updated?

File: db/schema.rb (hunk 3 above)
Comment (DevneyHamilton) [reply db/schema.rb]: thanks!

File: config/locales/en.yml (hunk 2 above)
Comment (hooverlunch) [reply config/locales/en.yml]: They will appear as new strings that need to be translated I believe. (If the key changes).

If the key doesn't change, not sure. Presumably Transifex should be smart enough to see there was a change to the value and prompt the translator to doublecheck?

File: app/models/broadcast.rb (hunk 1 above)
Comment (DevneyHamilton) [reply app/models/broadcast.rb]: yes!

File: config/locales/en.yml (hunk 2 above)
Comment (DevneyHamilton) [reply config/locales/en.yml]: updated keys in all the files

**Pull request merged**